app/src/main/java/skills/future/planer/ui/month/MonthFragment.java

```diff
@@ -4,67 +4,229 @@
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
-import android.widget.TextView;
 
 import androidx.annotation.NonNull;
+import androidx.core.content.ContextCompat;
 import androidx.fragment.app.Fragment;
 import androidx.lifecycle.ViewModelProvider;
+import androidx.lifecycle.ViewModelProviders;
 
+import com.prolificinteractive.materialcalendarview.CalendarDay;
+import com.prolificinteractive.materialcalendarview.DayViewDecorator;
+import com.prolificinteractive.materialcalendarview.DayViewFacade;
 import com.prolificinteractive.materialcalendarview.MaterialCalendarView;
 
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+import lombok.Getter;
+import lombok.Setter;
 import skills.future.planer.R;
 import skills.future.planer.databinding.FragmentMonthBinding;
-import skills.future.planer.ui.AnimateView;
+import skills.future.planer.db.task.TaskData;
+import skills.future.planer.db.task.TaskDataViewModel;
 
+/**
+ * MonthFragment represents a Month fragment.
+ *
+ * @author Paweł Helisz
+ * @version 0.2.2
+ * @since 0.2.1
+ */
+@Getter
+@Setter
 public class MonthFragment extends Fragment {
 
-    private TextView textView;
+    /**
+     * Calendar widget
+     */
     private MaterialCalendarView materialCalendarView;
-    private String date = "0"; //wybrany dzień z kalendarza
+
+    /**
+     * Month view model - extends ViewModel
+     */
     private MonthViewModel monthViewModel;
+
+    /**
+     * Databinding object
+     */
     private FragmentMonthBinding binding;
 
-    public MonthFragment() {
-    }
+    /**
+     * TaskData view model
+     */
+    private TaskDataViewModel mWordViewModel;
 
-    public String getDate() {
-        return date;
-    }
+    /**
+     * Hash map of tasks.
+     * Key - CalendarDay - day of a task
+     * Value - Integer - number of tasks in that day
+     */
+    private HashMap<CalendarDay, Integer> dayTasks;
+
+    /**
+     * Hash set of CalendarDay which fits into given category
+     */
+    private HashSet<CalendarDay> category1;
+    private HashSet<CalendarDay> category2;
+    private HashSet<CalendarDay> category3;
+    private HashSet<CalendarDay> category4;
 
-    public void setDate(String date) {
-        this.date = date;
+    /**
+     * Initialization of HashMap and HashSets
+     */
+    public MonthFragment() {
+        dayTasks = new HashMap<>();
+        category1 = new HashSet<>();
+        category2 = new HashSet<>();
+        category3 = new HashSet<>();
+        category4 = new HashSet<>();
     }
```

Comment:
dobre nazewnictwo tych pól

Reply:
gdybym tylko miał gotowe pobieranie kategorii dat...

Reply:
a co to ma wspólnego z nazewnictwem pól w klasie